packages/salesforcedx-vscode-core/src/generator/objectFileWatcher.ts

```diff
@@ -0,0 +1,155 @@
+import {
+  FauxClassGenerator,
+  GeneratorUtil
+} from '@salesforce/salesforcedx-sobjects-faux-generator/out/src/generator';
+import { EventEmitter } from 'events';
+import * as fs from 'fs';
+import * as path from 'path';
+import * as vscode from 'vscode';
+import { channelService } from '../channels';
+import { nls } from '../messages';
+import { notificationService } from '../notifications';
+import { SfdxProjectJsonParser } from '../util';
+
+const WAIT_TIME_IN_MS = 50;
+
```

Comment:
Thinking about generalizing and putting source file watcher stuff into a util for future use